docs/api/index.md

```diff
@@ -0,0 +1,55 @@
+
+This API is intended for use by public health organizations using Sara Alert, and thus Sara  Alert admins maintain a registered list of supported client applications. For access to a live production or demonstration environment, please contact system administrators at `sarasupport@aimsplatform.com` to discuss adding your client application to the approved list.
+
+# Security
+As indicated in the previous section, the API follows SMART on FHIR API standards, which includes support for both the [SMART App Launch](http://hl7.org/fhir/smart-app-launch/index.html) and [SMART Backend Services](https://hl7.org/fhir/uv/bulkdata/authorization/index.html) protocols. The SMART App Launch protocol is intended for use by user-facing apps, where access is granted based on a human user entering their existing Sara Alert username and password. The SMART Backend Services protocol is intended for backend services to autonomously use the Sara Alert API and access is granted based on a signed token. Detailed documentation of these two workflows can be found in the [Getting Started](api-getting-started) page.
+
+# What can the Sara Alert™ API do?
+
+At base, the API exists to allow other software systems to **read** and **write** Sara Alert data. Reading can mean either requesting specific data, e.g. "give me Monitoree X's information", or it can mean searching data, e.g. "give me the Monitoree with the e-mail address jane@example<span></span>.com". Writing data can mean putting new data into the system, e.g. "enroll this person as a new Monitoree", or it can mean updating existing data, e.g. "change this Monitoree's preferred reporting method to Opt-Out".
+
+The examples above are in terms of Monitorees, but the API also supports Symptom Reports and Lab Results. However, these entities are currently only supported for reading data.
+
+### For Monitorees
+**A client can read and write the following data elements:**<br>Workflow, First Name, Middle Name, Last Name, Date of Birth, Sex, White, Black or African American, American Indian or Alaskan Native, Asian, Native Hawaiian or Other Pacific Islander, Ethnicity, Primary Language, Interpretation Requirement, Address 1, Town/City, State, Zip, Country, Preferred Reporting Method, Preferred Contact Time, Primary Telephone Number, Secondary Telephone Number, E-mail Address, Last Date of Exposure, Symptom Onset Date, Monitoring Status
```

Comment:
This section is supposed to be geared towards less technical people, and so don't think pointing at the FHIR IG is ideal (plus the IG only indicates extensions, and doesn't include information about what fields we include that are natively supported in FHIR, e.g. address). The data dictionary would be better, but as of now the data dictionary doesn't have any indication of what is supported in the API.

Totally agree that it would be best to not have to update this list whenever it changes, but for now i think this might be the best we can do. Thoughts?

Reply:
I'm fine with it for now, especially since it's extremely easy to change. I think we should work towards getting an API data dictionary of sort out there just because I find this slightly harder to read/absorb.